Remove Masonry Align and Justify Tracks Test Cases

stwrt opened the pull request:
#### 16408a184624c8f2e25382a39498b09766f38335
<pre>
Remove Masonry Align and Justify Tracks Test Cases
<a href="https://bugs.webkit.org/show_bug.cgi?id=262720">https://bugs.webkit.org/show_bug.cgi?id=262720</a>
rdar://problem/116537788

Reviewed by Sammy Gill.

align-tracks and justify-tracks were removed from the spec.

Aligning WebKit WPT with upstream.
<a href="https://github.com/web-platform-tests/wpt/pull/42365">https://github.com/web-platform-tests/wpt/pull/42365</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/w3c-import.log: Removed.

Canonical link: <a href="https://commits.webkit.org/268958@main">https://commits.webkit.org/268958@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b811b0d397c45a1c0e62d345513bc52bec640e86

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/21123 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/21490 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/22187 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/23001 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/19642 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/24753 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/21685 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/20883 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/21347 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/21061 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/18316 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/23855 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/18219 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/19146 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/25417 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/19300 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/19348 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/23352 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/19889 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/16909 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/19169 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5070 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/23457 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/19747 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

sammygill reviewed (approved):
Looks good since we match upstream now. Note that there are still some tests in the fragmentation and baseline directories that still use this property. We should probably follow up on these too but it's not clear to me if they should just be removed or rebaselined to not use the property. Depends on how critical they are to what they are testing

stwrt committed:
Remove Masonry Align and Justify Tracks Test Cases
https://bugs.webkit.org/show_bug.cgi?id=262720
rdar://problem/116537788

Reviewed by Sammy Gill.

align-tracks and justify-tracks were removed from the spec.

Aligning WebKit WPT with upstream.
https://github.com/web-platform-tests/wpt/pull/42365

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-valid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-computed-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-computed.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-invalid-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-invalid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/justify-tracks-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-multi-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/masonry-align-tracks-stretch-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/align-tracks/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-multi-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/masonry-justify-tracks-stretch-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/justify-tracks/w3c-import.log: Removed.

Canonical link: https://commits.webkit.org/268958@main
